src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { sex, isMarried, partnerName, name, age } = person;
+
+  const spouseGender = sex === 'm'
+    ? 'wife'
+    : 'husband';
```

Comment:
It would be cool to keep such data in separate constants:
```js
const MALE = 'm';
const WIFE = 'wife';
const HUSBAND = 'husband';
...
const spouseGender = sex === MALE
  ? WIFE
  : HUSABAND;
```

Reply:
oh, I really didn't see it, sorry, I'll redo it now, thanks